src/components/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import icon from "../assets/images/night.png";
+import style from "./Header.module.scss";
+import iconMoon from "../assets/images/moon.png";
+import { useContext } from "react";
+import { CustomContext } from "../helpers/Context";
+
+function Header() {
+  const { isTheme = true, toggleTheme } = useContext(CustomContext);
```

Comment:
Please consider creating a custom hook for the specific context.

Reference https://kentcdodds.com/blog/how-to-use-react-context-effectively

Reply:
2,5 hours my friend, 2,5 hours!!!

Reply:
Like a turtle =)